### PR TITLE
Charges list

### PR DESCRIPTION
**Pull request by mliu356**
Made basic charges list for Payment Page 1, grabbing Activity names and prices from backend. We decided not to incorporate check box functionality into the charges list.

Charges list looks like:

screen shot 2019-02-17 at 5 50 34 pm

Mockup (charges list on bottom left):
screen shot 2019-02-17 at 5 50 57 pm

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -3,12 +3,13 @@ import logo from './logo.svg';
...
 import BFSNav from './components/Navbar';
 import PaymentPortal from './pages/ParentPortal'
+import PaymentPage from './pages/PaymentPage'
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">
+        {/* <header className="App-header">
```
Comment (SujeethJinesh) [inline client/src/App.js]: Let's revert this change and worry about replacing App.js later.

`git checkout -- client/src/App.js` to revert!

File: client/src/components/ChargesList.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import './PaymentContent.css';
+
+/**
+ * Content for page 0.
+ */
+class PaymentContent0 extends Component {
```
Comment (SujeethJinesh) [inline client/src/components/ChargesList.js]: Let's actually add some better and more descriptive naming to this so we won't be confused in the future.

File: client/src/components/PaymentContent.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#ActivityList{
+  /* height: 80px !important; */
```
Comment (SujeethJinesh) [inline client/src/components/PaymentContent.css]: Delete things that are commented

File: client/src/App.js (hunk 1 above)
Comment (mliu356) [reply client/src/App.js]: I ran that command but nothing seems to have changed?

Comment (Samyakkumar) [reply client/src/App.js]: I think you might need to run

<code> git checkout master -- client/src/App.js </code>

**Pull request merged**